# Generated for NaoQi method ALSoundDetection.setParameter
# Set detection parameters

# Name of the parameter. Please refer to the example below for the available parameters.
string paraDetect
# Value of the parameter.
float32 param
---


